packages/flutter/test/painting/text_painter_rtl_test.dart

```diff
@@ -9,6 +9,13 @@ import 'package:flutter_test/flutter_test.dart';
 const bool skipTestsWithKnownBugs = true;
 const bool skipExpectsWithKnownBugs = false;
 
+final String rloChar = String.fromCharCode(Unicode.RLO);
+final String lroChar = String.fromCharCode(Unicode.LRO);
+final String pdfChar = String.fromCharCode(Unicode.PDF);
+
+String lro(String s) => '${Unicode.LRO}L${s}L${Unicode.PDF}';
+String rlo(String s) => '${Unicode.RLO}R${s}R${Unicode.PDF}';
```

Comment:
This looks wrong — with the change to ints, these will print the code points in decimal rather than insert the actual characters.

I noticed this one because the lines appear in the diff (because they were moved from the bottom). But there may be other lines like this elsewhere too, both in the framework and in customer code. The breakage could be subtle, too, if the code isn't thoroughly tested. I think that raises the bar for what the benefit from changing these definitions' types would have to be in order to justify the churn.

(The reason this one isn't currently breaking CI on this PR is that these two helpers here in this test file are used only in a skipped test.)